### PR TITLE
DOCS-160 - Fix error in tpa upgrades section

### PR DESCRIPTION
**Pull request by djw-m**
## What Changed?

Error fixed
Page revamped for clarity (30%+ admonition text too much)
Some rephrasing for readability.

Please review entire page

### CODE REVIEW

File: product_docs/docs/pgd/5/upgrades/tpa_overview.mdx [hunk 1]
```diff
@@ -3,28 +3,32 @@ title: Upgrading PGD clusters with TPA
...
+If possible, always test upgrade processes in a QA environment first. This should help ensure that there are no unexpected factors that should be taken into account. TPA's ability to reproducibly deploy a PGD configuration makes it much easier to build a test environment to work with.
 !!!
 
 ## Minor and major PGD upgrades
 
-Minor version upgrades of PGD are managed automatically 
+TPA automatically manages minor version upgrades of PGD.
 
-Major version upgrades of PGD will require changes to the TPA `config.yml` file which contains the deployment configuration. 
+Major version upgrades of PGD will require changes to the TPA `config.yml` file, which contains the deployment configuration.
```
Comment (nidhibhammar) [inline product_docs/docs/pgd/5/upgrades/tpa_overview.mdx:23]:
```suggestion
Major version upgrades of PGD requires changes to the TPA `config.yml` file, which contains the deployment configuration. 
```

File: product_docs/docs/pgd/5/upgrades/tpa_overview.mdx [hunk 2]
```diff
@@ -3,28 +3,32 @@ title: Upgrading PGD clusters with TPA
...
 ## Minor and major PGD upgrades
 
-Minor version upgrades of PGD are managed automatically 
+TPA automatically manages minor version upgrades of PGD.
 
-Major version upgrades of PGD will require changes to the TPA `config.yml` file which contains the deployment configuration. 
+Major version upgrades of PGD will require changes to the TPA `config.yml` file, which contains the deployment configuration. 
 
-If you are upgrading to PGD 5, from previous PGD major versions, you will be able to use [`tpaexec reconfigure`](/tpa/latest/reference/tpaexec-reconfigure/) to help you make appropriate modifications to your deployment configuration. 
+When upgrading to PGD 5 from previous PGD major versions, you can use [`tpaexec reconfigure`](/tpa/latest/reference/tpaexec-reconfigure/). This command will help you make appropriate modifications to your deployment configuration.
```
Comment (nidhibhammar) [inline product_docs/docs/pgd/5/upgrades/tpa_overview.mdx:25]:
```suggestion
When upgrading to PGD 5 from previous PGD major versions, you can use [`tpaexec reconfigure`](/tpa/latest/reference/tpaexec-reconfigure/). This command helps you make appropriate modifications to your deployment configuration. 
```

File: product_docs/docs/pgd/5/upgrades/tpa_overview.mdx [hunk 3]
```diff
@@ -39,17 +43,23 @@ tpaexec upgrade clustername
 
 Where **clustername** is the name of the cluster and the path to the cluster configuration directory. By default, TPA will upgrade each node of the cluster to the latest minor versions of the software the nodes were configured with.
 
-!!! Note TPA's automated rolling upgrade procedure
 
-TPA will test first the cluster then the nodes. Each node will then be isolated from the cluster, upgraded and then returned to operation within the cluster.
+## TPA's automated rolling upgrade procedure
+
+TPA will first test the cluster and then the nodes.
```
Comment (nidhibhammar) [inline product_docs/docs/pgd/5/upgrades/tpa_overview.mdx:49]:
```suggestion
TPA first tests the cluster and then the nodes. 
```

File: product_docs/docs/pgd/5/upgrades/tpa_overview.mdx [hunk 4]
```diff
@@ -3,28 +3,32 @@ title: Upgrading PGD clusters with TPA
...
-Minor version upgrades of PGD are managed automatically 
+TPA automatically manages minor version upgrades of PGD.
 
-Major version upgrades of PGD will require changes to the TPA `config.yml` file which contains the deployment configuration. 
+Major version upgrades of PGD will require changes to the TPA `config.yml` file, which contains the deployment configuration. 
 
-If you are upgrading to PGD 5, from previous PGD major versions, you will be able to use [`tpaexec reconfigure`](/tpa/latest/reference/tpaexec-reconfigure/) to help you make appropriate modifications to your deployment configuration. 
+When upgrading to PGD 5 from previous PGD major versions, you can use [`tpaexec reconfigure`](/tpa/latest/reference/tpaexec-reconfigure/). This command will help you make appropriate modifications to your deployment configuration. 
 
 The `reconfigure` command will require settings for architecture (only `PGD_Always_ON` and PGD Proxy routing to run. Remember to back up your deployment configuration before running, and use the `--describe` and `--output` options to preview the reconfiguration.
```
Comment (nidhibhammar) [inline product_docs/docs/pgd/5/upgrades/tpa_overview.mdx:27]:
```suggestion
The `reconfigure` command requires settings for architecture (only `PGD_Always_ON` and PGD Proxy routing to run. Remember to back up your deployment configuration before running, and use the `--describe` and `--output` options to preview the reconfiguration. 
```

**Review by nidhibhammar — APPROVED** [review]
I left a few suggestions. The rest looks good to me.

**Comment by djw-m** [comment]
Most of it was the old copy reorged but yes, it needed freshening too.

**Pull request merged**